extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php

```diff
@@ -197,7 +197,7 @@ protected function getInfoboxData( $key ) {
 	protected function extractDataFromSource( \SimpleXMLElement $xmlNode ) {
 		$source = $this->getXmlAttribute( $xmlNode, self::DATA_SRC_ATTR_NAME );
 
-		return ( !empty( $source ) ) ? $this->getInfoboxData( $source )
+		return ( !empty( $source ) || $source == '0' ) ? $this->getInfoboxData( $source )
```

Comment:
Do you think a loose check like that is sufficient, or an explicit strict equality check for both 0 (int) and '0' (string) would be safer?